test-framework/junit5-internal/src/main/java/io/quarkus/test/QuarkusUnitTest.java

```diff
@@ -133,6 +135,18 @@ public QuarkusUnitTest setLogFileName(String logFileName) {
         return this;
     }
 
+    // set a Runnable that will run before ANYTHING else is done
+    public QuarkusUnitTest setBeforeAllCustomizer(Runnable beforeAllCustomizer) {
+        this.beforeAllCustomizer = beforeAllCustomizer;
```

Comment:
Why not use the `@BeforeAll` annotation in a static method?

Reply:
That's the first thing I tried :wink: . However that is executed _after_ the Quarkus extension and I needed it to execute before the build.